templates/experiment.html

```diff
@@ -19,7 +19,9 @@
 
 function disable(button) {
     const element = document.getElementById(button);
-    element.className = element.className.replace("primary", "inactive");
+    if (element !== null){
```

Comment:
It does happen, maybe some race condition? As part of the whole experiment procedure, I noticed sometimes the buttons do not work very well (this is also when the age starts giving exceptions)